tests/suites/0_stateless/09_0008_fuse_optimize_table.result

```diff
@@ -0,0 +1,15 @@
+4
+1
+5
```

Comment:
Magic numbers across two files, hard to review.

How about `select count(*) = 4  from fuse_history('db_09_0008', 't');`

Reply:
> How about select count(*) = 4 from fuse_history('db_09_0008', 't');

in `09_0008_fuse_optimize_table.sql`, change 

```
-- expects 4 history items, 3 of previous insertion, 1 for last compaction
select count(*) from fuse_history('db_09_0008', 't');
```

to 

```
-- expects 4 history items, 3 of previous insertion, 1 for last compaction
select count(*) = 4 from fuse_history('db_09_0008', 't');
````
,  and in `09_0008_fuse_optimize_table.result`, change the corresponding line, from `4` to `1`?

sure~

to keep the style in consistent, other `select count(*)`, even expects 1 will be changed to `select count(*) = ?` as well